python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -45,6 +48,14 @@ def __init__(self, registry: CollectorRegistry = None):
                 namespace="autoscaler",
                 registry=self.registry,
                 buckets=histogram_buckets)
+            self.update_time: Histogram = Histogram(
+                "update_time",
+                "Autoscaler update time. This is the time for an autoscaler"
```

Comment:
Super minor, but I think there needs to be a space at the end of this line

Reply:
thanks, nice catch!

Splitting strings between lines is the worst.